### DISCUSSION TITLE
graphql: productsCount

### DISCUSSION THREAD

**Discussion by soerenmeier** [craftcms/commerce#3646] [Ideas] [answered]
For entries there exists the query `entryCount` which can be useful for a search page or filter page.
It would be nice to have an equivalent function for products.

**Comment by nfourtythree** [comment, marked as answer]
Hi @soerenmeier 

Thank you for your message.

This is already possible with `productCount`. Example query:

```gql
{
  productCount(type: "bike")
}
```

Hopes this helps, thanks!

**Reply by soerenmeier** [reply]
Haha, now i feel dumb, thanks for the help.
Sorry this unnecessary post.
